backend/coins/btc/blockchain/blockchain.go

```diff
@@ -80,6 +80,11 @@ func (history TxHistory) Status() string {
 // ScriptHashHex is the hash of a pkScript in reverse hex format. Always 64 chars.
 type ScriptHashHex string
 
+// NewScriptHashHex creates the hash of an output's pubkeyScript.
```

Comment:
Is it only an output, though? what about a receiving address.

Reply:
removed `output's`.